department-of-veterans-affairs/va.gov-cms
# OpenAPI UI Hook to Event Subscriber

## Description

Closes #16456 by moving a library attachment via a hook to an event subscriber. 

## Testing done

Manually tested that "/admin/config/services/openapi" only has one JSON schema listed to explore and Swagger UI still works.

## Screenshots

<img width="1226" alt="Screenshot 2023-12-22 at 11 50 47 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/3640707/5645a6f7-2769-472c-ac9a-b9a768f1ac2f">


## QA steps

1. Log into to the CMS as a user who can get to "/admin/config/services/openapi".
2. Make sure only "VA.gov JSON:API" is listed under the Module column. JS hides two other options after the page loads that you'll probably see flicker while being removed.
3. Click "Explore with Swagger UI" and see that it still loads. 

### Definition of Done


### Select Team for PR review
